spec/models/authenticator/ldap_spec.rb

```diff
@@ -163,6 +164,20 @@ def authenticate
     let(:username) { 'alice' }
     let(:password) { 'secret' }
 
+    context "when using LDAP" do
+      let(:config) { super().merge(:ldap_role => true) }
```

Comment:
Yes, you need them if you want to call that way.

Not worth changing here but FWIW I avoid this by using another helper to change contexts when you're this far down the nesting, ie:

Instead of:
```
describe Thing do
  let(:credentials) { { username: "asdf", password: "qwerty" } }

  context "default do
    (test that uses "qwerty" password)
  end

  context "using a different password" do
    let(:credentials) { super().merge(password: "a different one") }
    (test that uses "a different one")
  end
end
```

I do
```
describe Thing do
  let(:credentials) { { username: "asdf", password: password } }
  let(:password) { "qwerty" }

  context "default do
    (test that uses "qwerty" password)
  end

  context "using a different password" do
    let(:password) { "a different one" }
    (test that uses "a different one")
  end
end
```

¯\\\_(ツ)_/¯